kubernetes/monitoring/services/udp-collector/udp-server.yaml

```diff
@@ -28,8 +14,13 @@ spec:
       labels:
         app: udp-server
```

Comment:
I suggest to expose metrics port for Prometheus scraping, see https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/das-server.yaml#L86 and use appropriate env too